### DISCUSSION TITLE
Devices disappear in Homekit

### DISCUSSION THREAD

**Discussion by njuart** [samemory/homebridge-eufy-security-control#278] [General]
Hi, recently at least 3 similar issues were reported. Honestly, it's a pain. Please, have a look why it may happen. Thx

https://github.com/samemory/homebridge-eufy-security/issues/277
https://github.com/samemory/homebridge-eufy-security/issues/275
https://github.com/samemory/homebridge-eufy-security/issues/264

**Comment by lenoxys** [comment]
please use issue. we won't provide support on discussion
